components/infobox/wikis/overwatch/infobox_person_player_custom.lua

```diff
@@ -182,8 +182,8 @@ end
 ---@return string?
 function CustomPlayer:createBottomContent()
 	if self:shouldStoreData(self.args) and String.isNotEmpty(self.args.team) then
-		local teamPage = Team.page(mw.getCurrentFrame(), self.args.team)
-		local team2Page = Team.page(mw.getCurrentFrame(), self.args.team2)
+		local teamPage = String.isNotEmpty(self.args.team) and Team.page(mw.getCurrentFrame(), self.args.team) or nil
```

Comment:
```suggestion
		local teamPage = Team.page(mw.getCurrentFrame(), self.args.team)
```
This check is on the previous line no?

Reply:
yes, apparently i'm pretty blind in recent weeks